snakemake/scheduler.py

```diff
@@ -194,11 +194,25 @@ def __init__(
                 cores=local_cores,
                 keepincomplete=keepincomplete,
             )
-            # we need to adjust the maximum status checks on a
-            # SLURM cluster for not to overstrain the scheduler
-            if max_status_checks_per_second > 1:
-                # # every 30 sec is a resonable default
-                max_status_checks_per_second = 0.03
+            # we need to adjust the maximum status checks per second
+            # on a SLURM cluster, to not overstrain the scheduler;
+            # timings for tested SLURM clusters, extracted from --verbose
+            # output with:
+            # ```
+            #   grep "sacct output" .snakemake/log/2023-02-13T210004.601290.snakemake.log | \
+            #   awk '{ counter += 1; sum += $6; sum_of_squares += ($6)^2 } \
+            #     END { print "average: ",sum/counter," sd: ",sqrt((sum_of_squares - sum^2/counter) / counter); }
```

Comment:
This is quick code to get average and standard deviation of `sacct` timings that this PR introduces (along with the respective `scontrol` timings) into the debugging output. If others could use it to also provide timings, this could help decide whether `scontrol` is the better default for checking the status of `active_jobs`.